src/OpenTelemetry.Api/README.md

```diff
@@ -358,12 +358,13 @@ to be associated with `Activity`. There is no `Status` class in .NET, and hence
 Example:
 
 ```csharp
-activity?.SetTag("otel.status_code", 2);
+activity?.SetTag("otel.status_code", "Error");
 activity?.SetTag("otel.status_description", "error status description");
 ```
 
-StatusCodes can be 0, 1, 2 which corresponds to `Unset`, `Ok` and `Error`
-respectively from [StatusCode.cs](./Trace/StatusCode.cs)
+Values for the StatusCode tag have to be the strings "Unset", "Ok", or "Error",
```

Comment:
```suggestion
Values for the StatusCode tag have to be the strings "Unset", "Ok", or "Error",
```
Values for the StatusCode tag must be one of the strings "Unset", "Ok", or "Error",